app/components/containerInspect/containerInspect.html

```diff
@@ -0,0 +1,21 @@
+<rd-header>
+  <rd-header-title title="Container inspect">
+  </rd-header-title>
+  <rd-header-content>
+    <a ui-sref="containers">Containers</a> &gt; <a ui-sref="container({id: data.Id})">{{ ::data.Name|trimcontainername }}</a> &gt; Inspect
+  </rd-header-content>
+</rd-header>
+
+<div class="row">
+  <div class="col-lg-12 col-md-12 col-xs-12">
+    <rd-widget>
+      <rd-widget-header icon="fa-icon-circle" title="Inspect">
+      </rd-widget-header>
+      <rd-widget-body>
+        <pre>
+          {{ containerInfo|json:4 }}
```

Comment:
Try to remove the spacing here, otherwise it will be displayed in the rendered HTML.

```html
<rd-widget-body>
  <pre>{{ data | json:4 }}</pre>
</rd-widget-body>
```

Reply:
Well, that's embarrassing 🤦‍♂️ 
I was trying to find what causes additional spaces at the start for a relatively big amount of time and completely forgot that HTML count spacing in element too.